src/store/modules/app/walletConnect.js

```diff
@@ -118,9 +124,9 @@ export const approveSession = async ({ dispatch }, { proposal, accounts }) => {
   console.log('topic', topic)
   const session = await acknowledged()
   console.log('approvedSession', session)
-  dispatch('getPairings')
-  dispatch('getSessions')
-  dispatch('getSessionProposals')
+  await dispatch('getPairings')
```

Comment:
check if we don't need it because all actions are independent and we should not wait for all of them